compiler/rustc_codegen_cranelift/src/intrinsics/llvm_x86.rs

```diff
@@ -463,7 +463,7 @@ pub(crate) fn codegen_x86_llvm_intrinsic_call<'tcx>(
             // into 0x80000000 for which Cranelift doesn't have a native instruction.
             codegen_inline_asm_inner(
                 fx,
-                &[InlineAsmTemplatePiece::String(format!("cvttps2dq xmm0, xmm0"))],
+                &[InlineAsmTemplatePiece::String(format!("cvttps2dq xmm0, xmm0").into())],
```

Comment:
shouldn't `format!` be removed as the format string doesn't contain any placeholders

Reply:
Indeed, done.